modular_skyrat/modules/skyrat_access_helpers/accesshelpers.dm

```diff
@@ -0,0 +1,94 @@
+/* Read the main access_helpers.dm for more information on these mapping tools.
+TLDR:
+Any requires EITHER OR access defined on the ID
+All requires THAT specific access defined on the ID (so don't put more than one on a door at a time UNLESS ITS INTENTIONAL for a job to have all the access(es))
+For mapping examples, look at /TG/s MetaStation
+*/
```

Comment:
Common formatting for multi-line comments. Check out this handy extension, if you don't have it already:
https://marketplace.visualstudio.com/items?itemName=stylemistake.auto-comment-blocks
```suggestion
/** Read the main access_helpers.dm for more information on these mapping tools.
 * TLDR:
 * Any requires EITHER OR access defined on the ID
 * All requires THAT specific access defined on the ID (so don't put more than one on a door at a time UNLESS ITS INTENTIONAL for a job to have all the access(es))
 * For mapping examples, look at /TG/s MetaStation
*/
```